### PR TITLE
Non-trivial merge to 0.23

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

**Comment by bplommer** [comment]
Just re-triggered it and same deal. Any idea what's going on here?

**Comment by armanbilge** [comment]
Ah, I know what's going on. The ci thing which segregates JVM/JS builds in the matrix went missing. So now it's trying to do both at the same time and Scala.js linking step is _extremely_ memory-intensive. So GH is killing the job.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,27 +77,27 @@ jobs:
 
       - name: Check formatting
         if: matrix.scala != '3.1.0'
-        run: sbt ++${{ matrix.scala }} '${{ matrix.ci }}' scalafmtCheckAll
```
Comment (armanbilge) [inline .github/workflows/ci.yml:68]: This `${{ matrix.ci }}` is what was used to to set the root project to either `rootJVM` or `rootJS`.

Comment (bplommer) [reply .github/workflows/ci.yml:68]: I doubt Ross would mind if you felt like pushing a fix 😄

Comment (rossabaker) [reply .github/workflows/ci.yml:68]: Oh, something in the merge made it look like that had been intentionally removed, and I didn't understand it, so I played along.

**Comment by rossabaker** [comment]
Nice green checkmark for a build so broken that didn't run at all.

**Review by bplommer — APPROVED** [review]

**Pull request merged**